pkg/cloudprovider/gce/gce.go

```diff
@@ -479,6 +479,17 @@ func (gce *GCECloud) UpdateTCPLoadBalancer(name, region string, hosts []string)
 			return err
 		}
 	}
+
+	// Try to verify that the correct number of nodes are now in the target pool.
```

Comment:
I think this may be a case of being more cautious than we need about a bug that we think has been fixed, but @brendandburns is pretty strongly in favor of making the logging here to make bug detection and debugging significantly easier in case there are any other issues in the logic.